scalableminds/webknossos
# Unify Backend Enumerations

### URL of deployed dev instance (used for testing):
- https://unifybackendenums.webknossos.xyz

### Steps to test:
- Enum fields should still be intact: list task types, create, edit allowed modes
- Should be able to annotate, save, download, try hybrid, skeleton, volume
- CI should run through

### Issues:
- fixes #5186 

------
